### PR TITLE
Build warnings: includes, byline and more

### PR DESCRIPTION
**Pull request by Garneauma**
Resolving build warnings part 4:
- Fixing byline file include which did not recognize "i18nText-lang"
- Fixing syntax for "byline" include call
- Fixing syntax and url for "placeholdercontent" include call
- Integrating "comp-download" include content directly in files calling it (2 files)
- Integrating "minister-port" include content directly in files calling it (2 files)
- Integrating "fullmenu" include content directly in files calling it (2 files)
- Integrating "carousel-small" include content directly in files calling it (2 files)

### CODE REVIEW

**Comment by Ricokola** [comment]
Pre-approved upon successful testing

File: sites/gcweb-menu/ajax/sitemenu-fr.hbs [hunk 1]
```diff
@@ -3,4 +3,124 @@
 	language: "fr"
 }
 ---
-{{>fullmenu}}
+<!-- secondary navigation if enabled -->
+<div class="pnl-strt container nvbar">
+	<h2 class="wb-inv">Sujets</h2>
```
Comment (duboisp) [inline sites/gcweb-menu/ajax/sitemenu-fr.hbs:8]: I did found "Menu des sujets"

ref. https://github.com/wet-boew/themes-dist/blob/v9.0.0-gcweb/ajax/sitemenu-fr.html#L8

File: sites/gcweb-menu/ajax/sitemenu-en.hbs [hunk 2]
```diff
@@ -3,4 +3,124 @@
 	language: "en"
 }
 ---
-{{>fullmenu}}
+<!-- secondary navigation if enabled -->
+<div class="pnl-strt container nvbar">
+	<h2 class="wb-inv">Topics</h2>
```
Comment (duboisp) [inline sites/gcweb-menu/ajax/sitemenu-en.hbs:8]: I did found "Topics menu"

ref: https://github.com/wet-boew/themes-dist/blob/v9.0.0-gcweb/ajax/sitemenu-en.html#LL8C20-L8C34

**Review by duboisp — CHANGES_REQUESTED** [review]
Other than the heading text that are not matching for the old mega menu, the rest which I tested and reviewed look good.

**Comment by Garneauma** [comment]
> Other than the heading text that are not matching for the old mega menu, the rest which I tested and reviewed look good.

The headings have been updated. :)

**Review by duboisp — APPROVED** [review]
Completed the review + local testing. And the requested change was completed as expected.

**Pull request merged**